src/html.js

```diff
@@ -31,6 +38,12 @@ export function generateHTML(context){
     // outer
     context.outer = wrapElement(context.overflow);
     context.outer.classList.add(CLASSES.outer);
+
+    // theme: corner
+    if(context.isCornerTheme){
+        context.outer.classList.add(CLASSES.master + '-corner');
```

Comment:
Do not use `CLASSES.master + '-corner'`. Please create `CLASSES.cornerMaster` in the config.